docs/core/install/linux-ubuntu.md

```diff
@@ -1,200 +1,181 @@
 
-[!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
+> [!WARNING]
+> It's recommended that you avoid using both feeds for .NET, as this leads to problems when apps try to resolve a specific version of .NET. Mixing the package feeds for .NET is an advanced scenario, and is covered in [Troubleshoot .NET errors related to missing files on Linux - Solutions](linux-package-mixup.md).
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+| Method | Pros | Cons |
+|--------|------|------|
+| [Package manager<br>(Microsoft feed)*](#register-the-microsoft-package-repository) | <ul><li>Supported versions always available.</li><li>Preview releases are available.</li><li>Patches are available right way.</li><li>Dependencies are included.</li><li>Easy removal.</li></ul> | <ul><li>Requires registering the Microsoft package repository.</li><li>Only supports x64 Ubuntu.</li></ul> |
+| [Package manager<br>(Ubuntu feed)](#supported-distributions) | <ul><li>Usually the latest version is available.</li><li>Patches are available right way.</li><li>Dependencies are included.</li><li>Easy removal.</li></ul> | <ul><li>.NET versions available vary by Ubuntu version.</li><li>Preview releases aren't available.</li><li>Only supports x64 Ubuntu.</li></ul> |
+| [Snap](linux-snap.md) | <ul><li>Dependencies are included.</li><li>All supported CPU architectures available.</li><li>Easy to update to the latest patch version.</li><li>Easy removal.</li></ul> | <ul><li>Cumbersome to install and enable multiple runtimes.</li></ul> |
+| [Script \ Manual extraction](linux-scripted-manual.md) | <ul><li>Control where .NET is installed.</li></ul> | <ul><li>Manually install updates.</li><li>Manually install dependencies.</li><li>Manual removal.</li></ul> |
 
-## Supported distributions
+\* This is the recommended installation method for end-users and developers.
 
-The following table is a list of currently supported .NET releases and the versions of Ubuntu they're supported on.
+## Decide how to install .NET
 
-| Ubuntu                 | .NET       |
-|------------------------|------------|
-| [22.10](#2210)         | 7, 6       |
-| [22.04 (LTS)](#2204)   | 7, 6       |
-| [20.04 (LTS)](#2004)   | 7, 6       |
-| [18.04 (LTS)](#1804)   | 7, 6       |
-| [16.04 (LTS)](#1604)   | 6          |
+I want to install .NET because...
 
-[!INCLUDE [versions-not-supported](includes/versions-not-supported.md)]
+- **I want to create a .NET app:**
 
-## Install preview versions
+  We recommend you use APT and the Microsoft package repository. For more information, see the [Register and install with the Microsoft package repository](#register-the-microsoft-package-repository) section.
```

Comment:
I do not understand why we recommend Microsoft package repository for dev environment, but Ubuntu package repository for production environment. Why is that?

Reply:
I was going under the assumption that devs may be installing multiple versions over time, which means it's possible those versions aren't available in the ubuntu repository. Also, people run into issues with dev tools not detecting .NET, so I would think it's quite possible that this audience gets into a package source mixup quicker than others.

This suggestion was to avoid that.

Reply:
Any thoughts on my reasoning about this?

Reply:
In general, it is better to use the same bits for both dev and production environment. It reduces chances of running into issues that reproduce in one environment and not in the other environment.

I think the doc should go more into pros vs. cons why you may want to do something, instead of just providing blanket statements. Similar to what I have said in https://github.com/dotnet/docs/pull/34234#discussion_r1123999600.